frontend/ui/src/pages/Channels/Providers/Airy/ChatPlugin/sections/ConnectNewChatPlugin.tsx

```diff
@@ -1,36 +1,20 @@
-import React, {useEffect, useState} from 'react';
+import React, {useState} from 'react';
 
 import {Button, Input} from 'components';
 
 import styles from './ConnectNewChatPlugin.module.scss';
 
-import {
-  cyChannelsChatPluginConnectButton,
-  cyChannelsChatPluginFormNameInput,
-  cyChannelsChatPluginFormSubmitButton,
-} from 'handles';
+import {cyChannelsChatPluginFormNameInput, cyChannelsChatPluginFormSubmitButton} from 'handles';
 
```

Comment:
@pascal-airy we removed the connect button so its cypress test got deleted as well